dev/com.ibm.ws.jdbc/src/com/ibm/ws/rsadapter/impl/WSRdbManagedConnectionImpl.java

```diff
@@ -1160,15 +1191,19 @@ public final boolean isGlobalTransactionActive() {
     }
 
     /**
-     * This method checks if transaction enlistment is enaabled on the MC
+     * This method checks if transaction enlistment is enabled on the MC
+     * and indicates whether or not this managed connection should enlist 
+     * in application server managed transactions.
      * 
-     * @return true if transaction enlistment is enabled, false otherwise.
+     * @return true if transaction enlistment is enabled and supported, false otherwise.
      */
     public final boolean isTransactional() {
         // Take a snapshot of the value with first use (or reuse from pool) of the managed connection.
         // This value will be cleared when the managed connection is returned to the pool.
         if (transactional == null) {
             transactional = mcf.dsConfig.get().transactional;
+            if (useDriverIsoLvl && transactional)
```

Comment:
I have some concerns here about switching the transactional value.  If it is only true because that is the default that I think it's okay to switch the value, but if they have configured TRANSACTON_NONE and transactional=true in their config it is probably better to reject that configuration rather than switch the value under the covers.

Reply:
It would probably be appropriate to use ignorewarnfail if they have TRANSACTION_NONE configured with transactional=true

Reply:
Also, I would like to see this check moved from this location to the point where we accept the configuration, in which case we would simply compare the values of the transactional and isolationLevel config properties.